### PR TITLE
Fix/terminal wrap

### LINKED ISSUE #149: Speedway CLI Schema Creation Breaks on Terminal
* **Version**: v1.0.3-beta.1 
* **Platform**: MacOS
* **Subsystem**:

#### Issue description
Speedway CLI schema creation breaks on integrated VSCode terminal, iterm2 and Apple terminal.

#### Steps to reproduce the issue

1. Open a terminal 
2. cd into speedway/cmd/speedway-cli/
3. Run `go run main.go schema create`
4. Add label and start typing in a long list of labels:types
5. Witness

#### What's the expected result?

- Schema gets created

#### What's the actual result?

- New line of the command is printed every time a character is added to the list after a certain string size (which i assume is relative to the width of the terminal) is passed

#### Additional details / screenshot

- Screen Shot 2022-09-17 at 6 09 26 PM

### PR DESCRIPTION
**Pull request by 6missedcalls**
Refactored all prompts to fix terminal wrapping bug

Fixed input for create schema (CLI)

Former Input:
``` name:string,age:int ```

Now Accepted Inputs:
``` name:string,age:int ```
``` name:string, age:int ```
``` name:string , age:int ```

## Changes

- internal/prompts

### New Features *(required)*
N/A

### Deprecations *(required)*
N/A

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes #149 #150 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: cmd/speedway-cli/speedwaycmd/object/getobject.go [hunk 1]
```diff
@@ -39,28 +39,16 @@ func BootstrapGetObjectCommand(ctx context.Context, logger *golog.Logger) (getOb
...
-				cidPrompt := promptui.Prompt{
-					Label: "Enter Object CID",
-				}
-				cid, err = cidPrompt.Run()
-				if err != nil {
-					fmt.Printf("Command failed %v\n", err)
-					return
-				}
+				cid = (&prompter.Prompter{
+					Message: "Enter the CID of the object you want to query",
```
Comment (mcjcloud) [inline cmd/speedway-cli/speedwaycmd/object/getobject.go:43]: These prompts are too verbose in my opinion. I think prompts along the lines of "Object CID: " are better

**Review by joshLong145 — APPROVED** [review]

File: cmd/speedway-cli/speedwaycmd/object/builder.go [hunk 2]
```diff
@@ -6,8 +6,8 @@ import (
 	"math"
```
Comment (ntindle) [inline cmd/speedway-cli/speedwaycmd/object/builder.go:6]: Open an issue to swap to schema docusment?

File: cmd/speedway-cli/speedwaycmd/object/builder.go [hunk 3]
```diff
@@ -33,33 +33,30 @@ func BootstrapBuildObjectCommand(ctx context.Context, logger *golog.Logger) (bui
...
-				schemaDidPrompt := promptui.Prompt{
-					Label: "Enter Schema DID",
-				}
-				schemaDid, err = schemaDidPrompt.Run()
-				if err != nil {
-					fmt.Printf("Command failed %v\n", err)
+				schemaDid = (&prompter.Prompter{
+					Message: "Schema DID",
+				}).Prompt()
+				if schemaDid == "" {
```
Comment (ntindle) [inline cmd/speedway-cli/speedwaycmd/object/builder.go:57]: should probably have a DID validator run here

File: cmd/speedway-cli/speedwaycmd/registry/Login.go [hunk 4]
```diff
@@ -22,29 +22,21 @@ func bootstrapLoginCommand(ctx context.Context, logger *golog.Logger) (loginCmd
...
-				fmt.Printf("Command failed %v\n", err)
-				return
-			}
+			addr := (&prompter.Prompter{
+				Message: "Enter the address of the registry you wish to login to",
+			}).Prompt()
+
+			password := (&prompter.Prompter{
+				Message: "Enter your password",
+				Regexp:  regexp.MustCompile(`.{8,}`),
```
Comment (ntindle) [inline cmd/speedway-cli/speedwaycmd/registry/Login.go:31]: Don't think this regex is the same as the ones for creating an account

**Review by ntindle — APPROVED** [review]

**Pull request merged**